vital/algo/integrate_depth_maps.cxx

```diff
@@ -26,16 +24,14 @@ ::integrate_depth_maps()
 }
 
 /// Integrate multiple depth maps into a common volume
-
-/**
- *
- * \param [in]     minpt_bound the min point of the bounding region
- * \param [in]     maxpt_bound the max point of the bounding region
- * \param [in]     depth_maps  the set of floating point depth map images
- * \param [in]     cameras     the set of cameras, one for each depth map
- * \param [in,out] volume      the fused volumetric data
- * \param [out]    spacing     the spacing between voxels in each dimension
- */
+///
+///
+/// \param [in]     minpt_bound the min point of the bounding region
+/// \param [in]     maxpt_bound the max point of the bounding region
+/// \param [in]     depth_maps  the set of floating point depth map images
+/// \param [in]     cameras     the set of cameras, one for each depth map
+/// \param [in,out] volume      the fused volumetric data
+/// \param [out]    spacing     the spacing between voxels in each dimension
```

Comment:
There is an extra comment line here, but it is because it is in the original as well.

Reply:
Yeah... this should be "harmless" (read: won't generate wrong documentation, won't "paint us into a corner" by making future automated clean-up harder or impossible). We should probably plan to (in a future PR) clean up:
- Trailing space after `///` comments (partly because it will make the other clean-ups easier)
- Consecutive empty `///` comments
- Leading or trailing empty `///` comments

Reply:
Also, remove blank lines between separator lines (`^\s*// ?[-=]+$`) and `///` comments.